plutus-core/plutus-core/src/PlutusCore/Crypto/Hash.hs

```diff
@@ -21,6 +24,14 @@ sha2_256 = digest (Proxy @SHA256)
 sha3_256 :: BS.ByteString -> BS.ByteString
 sha3_256 = digest (Proxy @SHA3_256)
 
+-- | Hash a [[BSL.ByteString]] using the Blake2B-224 hash function.
```

Comment:
Pardon my ignorance, what is this `[[...]]` syntax in Haddock?

Reply:
Ah, I asked about that before and no-one could tell me, then I just copied it without looking.